app/views/exception_hunter/errors/show.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="row statistics-row">
+  <div class="column column-20"></div>
+  <div class="column column-20"></div>
+  <div class="column column-20"></div>
+  <div class="column column-20"></div>
+  <div class="column column-20">
+  <%= render partial: 'exception_hunter/errors/pagy/pagy_nav', locals: {pagy: @pagy} %>
+  </div>
+</div>
+
+<div class="row error-row error-row--header">
+  <div class="column column-35">Name</div>
+  <div class="column column-25">Message</div>
+  <div class="column column-25"></div>
+  <div class="column column-15">Ocurred at</div>
+</div>
+
+
+<div class="row error-row-no-border">
+  <div class="column column-35 error-cell">
+    <%= @error.first.error_group.error_class_name %>
+  </div>
+  <div class="column column-40 error-cell">
+    <%= @error.first.message %>
```

Comment:
Why first? This should be just one error right?

Reply:
pagy returns an array on `@error`